decimal64scan.go

```diff
@@ -36,6 +36,7 @@ func MustParseDecimal64(s string) Decimal64 {
 
 // Scan implements fmt.Scanner.
 func (d *Decimal64) Scan(state fmt.ScanState, verb rune) error {
+	*d = SNaN64
```

Comment:
Don't change it if returning an `error`.

Reply:
default behaviour from any bad input is SNaN64 though which is why I set the pointer at the start